### PR TITLE
Adds Gondola meat

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
add: Killing gondolas now lets you harvest meat from them. Eating it raw might be a bad idea.
/:cl:

I've heard of this story where eating mermaid flesh either makes you immortal or gives you a horrible fate, so I thought, why not apply this to our beloved gondolas?

Gondola meat contains a reagent that causes the gondola transformation disease - it will continuously dose the victim with pax until eventually triggering the transformation. The cure? Condensed capsaicin.

I've reduced the default stage_prob (translates to progress speed) and changed the cure to condensed capsaicin, so you can't easily fuck over people with tranquillity smoke grenades.

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1829,3 +1829,15 @@
 	if(prob(30))
 		to_chat(M, "You should sit down and take a rest...")
 	..()
+
+/datum/reagent/tranquility
```
Comment (XDTM) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: tranquillity

**Comment by davethwave** [comment]
Bath salts? Yikes, that's a hard thing to make! How long does the transformation take anyways?

**Comment by 81Denton** [comment]
I picked bath salts since themetically they'd be the best thing to beat tranquillity out of you. I guess something like condensed capsaicin (pepper spray) would be better gameplay wise?

The duration is more of less random, but I've had up to 5+ minutes in some cases?

**Comment by RandomMarine** [comment]
Have you taken into account that someone can grind out the tranquility and syringegun/chemnade it onto people?

**Comment by 81Denton** [comment]
Yeah, hence the low `stage_prob` ->progress speed. I'll also change the cure to condensed capsaicin, so it's not like the slime mutation toxin where you need something absurdly rare like frost oil as a cure.

**Comment by Militaires** [comment]
i approve, also make it sell cargo for a high price

**Comment by SpaceManiac** [comment]
Tranquility is spelled with one L in American and therefore Space English (see: Sea of Tranquility, Tranquility Base, the Tranquility ISS module). For extra precedent, lightgeist `desc` uses one L.

Works in testing though.

**Comment by 81Denton** [comment]
Oh thank god, I could swear I always spelled it with one L

**Review by SpaceManiac — APPROVED** [review]
Now that's what I call efficiency

**Comment by XDTM** [comment]
TIL

File: code/modules/food_and_drinks/food/snacks/meat.dm [hunk 2]
```diff
@@ -304,6 +314,10 @@
 	tastes = list("meat" = 1, "rock" = 1)
 	foodtype = MEAT
 
+/obj/item/reagent_containers/food/snacks/meat/steak/gondola
+	name = "gondola steak"
+	tastes = list("meat = 1", "tranquility" = 1)
```
Comment (mattroks101) [inline code/modules/food_and_drinks/food/snacks/meat.dm]: I'm late but shouldn't that be `"meat" = 1` and not `"meat = 1"`

**Pull request merged**